### DISCUSSION TITLE
Feature Request : pinecone or milvus

### DISCUSSION THREAD

**Discussion by revbin** [danny-avila/LibreChat#373] [Feature Requests & Suggestions]
Hi

its is possible to add pinecone or milvus to the script as AI memory where we can upload our vectored document to the DB .

regards

**Comment by danny-avila** [comment]
We are releasing a Plugins endpoint this week which will utilize plugins. There is still work to do for file integration, and a retrieval plugin still needs to be made, but yes this is planned for now.
